Readability.js

```diff
@@ -1678,15 +1678,18 @@ Readability.prototype = {
    *
    * @return boolean Whether or not we suspect parse() will suceeed at returning an article object.
    */
-  isProbablyReaderable: function() {
-    var nodes = this._doc.getElementsByTagName("p");
-    if (nodes.length < 5) {
-      return false;
-    }
-
-    var possibleParagraphs = 0;
+  isProbablyReaderable: function(helperIsVisible) {
```

Comment:
I wish we could have `helperIsVisible` part of the API, but I suspect this is impossible to achieve relying on JSDOMParser… 

Though for clients relying on a real DOM impl., maybe we could define some fallback? I've been using [this](https://github.com/n1k0/casperjs/blob/11bb84995fd543440891a14baff5b8e1bb9b7866/modules/clientutils.js#L156-L171) in casperjs with good reliability so far.